Funcky.Analyzer/Funcky.Analyzer.Vsix/Funcky.Analyzer.Vsix.csproj

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
```

Comment:
This project should probably disable packing

Reply:
Auto generated.

Reply:
It should still disable packing, or do we want to publish this as a NuGet package?

Reply:
Set IsPackable to false: a3ce278ffaa3c986291d49f827e8cdb91a08d644